backends/gdx-backend-robovm/src/com/badlogic/gdx/backends/iosrobovm/IOSApplication.java

```diff
@@ -433,6 +433,11 @@ public void setContents (String content) {
 				UIPasteboard.getGeneralPasteboard().setString(content);
 			}
 
+			@Override
+			public boolean hasContents () {
+				return UIPasteboard.getGeneralPasteboard().hasStrings();
```

Comment:
This seems iOS10+, I think we need an OS check here (we support up to minOS 9)

Reply:
Oh @obigu, you are right, thank you for spotting this!

I will push a fix.

Reply:
Thanks!